### ISSUE TITLE
A

### ISSUE THREAD

**Issue by giik0n** [giandifra/arcore_flutter_plugin#224]

**Comment by giik0n** [comment]
Hello, do u have camera feed in application? after upgrading flutter its stops working and look like empty screen, maybe the problem is in something another (compileSDKversion).
Did u had something simmilar?

**Comment by jahanzaib999** [comment]
I am having the same issue did you find the solution?

**Comment by giik0n** [comment]
> I am having the same issue did you find the solution?

No, i just switched the library, and currently using model_viewer_plus
